src/darker/linting.py

```diff
@@ -75,7 +78,7 @@ def run_linter(cmdline: List[str], git_root: Path, paths: Set[Path]) -> None:
     )
     # assert needed for MyPy (see https://stackoverflow.com/q/57350490/15770)
     assert linter_process.stdout is not None
-    edited_linenums_differ = EditedLinenumsDiffer(git_root)
+    edited_linenums_differ = EditedLinenumsDiffer(git_root, revision)
```

Comment:
Do you want to make the revision mandatory in EditedLinenumsDiffer ?

Reply:
Thanks, that's indeed a good idea.